media/css/firefox/new/scene2_email.scss

```diff
@@ -120,103 +121,43 @@
 
 
 // Newsletter form
-.c-newsletter {
-    @include font-size-level6;
-
-    a:link,
-    a:visited {
-        color: #fff;
-
-        &:hover,
-        &:focus {
-            color: #eee;
-        }
-    }
+.mzp-c-newsletter {
+    @include bidi(((text-align, left, right),));
+    display: block;
+    padding: 0;
 
-    input[type='email'],
-    select {
-        width: 100%;
+    .mzp-c-newsletter-form {
+        padding: 0;
     }
 
-    input[type='email'] {
-        @include border-box;
-        background: #fff;
-        border-radius: 3px;
-        border: 1px solid #4a4a4f;
-        display: inline-block;
-        line-height: 1.25;
-        padding: .25em 8px;
-
-        &:hover {
-            border-color: #000;
-        }
+    .mzp-c-button {
+        box-shadow: none;
+        background: $color-black;
+        border: 0;
+        color: $color-white;
+        transition: box-shadow 150ms, background-color 150ms, color 150ms;
 
+        &:hover,
         &:focus {
-            border-color: #000;
-            box-shadow: 0 0 0 2px #45a1ff;
-        }
-    }
-
-    .field {
-        display: block;
-        margin-bottom: .75em;
-    }
-
-    label {
-        display: block;
-        font-weight: bold;
-        margin: 0 0 .25em;
-    }
-
-    .privacy-check-label {
-        @include font-size-small;
-        padding-left: 2em;
-        display: block;
+            background: $color-white;
+            box-shadow: 4px 4px 0 0 #000;
```

Comment:
Buttons are always a little contentious, but we seem to be making quite a few button related tweaks on Protocol pages that have not yet been adopted in the design system. Being forward thinking is good, but the more we do this the more papercuts we create for ourselves in the future to go back and undo all these custom styles. I guess this is up to our designers, but imho the UX here is not going to be negatively impacted if we just use the stock button for now :/

Reply:
I'm working on a PR to make this the base button style in Protocol. But yeah, this page would probably be fine with the current Protocol button until it's updated in Protocol (at which point this would automatically update once the new button is published upstream; the system works!) I'll remove this extra styling.

Reply:
Oh now I remember... the default button switches to white in a dark theme but Jennifer wanted it to be black here so it would be distinct from the form fields. So since I needed to invert the color anyway that's why I added the shadow... so it seems this still needs to be a non-standard button for the moment, but I'll at least remove the shadow and just invert the colors. 

And I'll make sure to include a theme class when I update the buttons in Protocol so we can still have a black button in a dark theme when needed.